services/src/main/java/org/exoplatform/addons/gamification/entities/domain/effective/GamificationActionsHistory.java

```diff
@@ -194,6 +194,12 @@
             " AND g.receiver = :receiverId" +
             " AND g.objectId = :objectId"
 )
+
+@NamedQuery(
+    name = "GamificationActionsHistory.getRealizationsByObjectId",
+    query = "SELECT g FROM GamificationActionsHistory g" +
+            " WHERE g.objectId = :objectId"
```

Comment:
We need the object type(s) here as well, isn't it ?

```suggestion
            " WHERE g.objectId = :objectId"
```